### PR TITLE
Enable Retina display resolution

### PR DESCRIPTION
**Pull request by chenasraf**
Not sure why this wasn't in from the get-go! Dragging the app to [Retinizer](http://retinizer.mikelpr.com/) makes it look amazing on Retina, and this change should make that work for everyone without having to do anything extra.

### CODE REVIEW

File: Messenger/Info.plist [hunk 1]
```diff
@@ -32,5 +32,7 @@
 	<string>MainMenu</string>
 	<key>NSPrincipalClass</key>
 	<string>FBMApplication</string>
+	<Key>NSHighResolutionCapable</key>
+	<String>True</string>
```
Comment (grEvenX) [inline Messenger/Info.plist:36]: Bad casing on XML keys, `<Key>` => `<key>` and `<String>` => `<string>`

**Comment by rsms** [comment]
Weird. It does render in retina for me without `NSHighResolutionCapable`. Apple says that the `NSHighResolutionCapable` key is only useful for apps which aren't already retina-enabled — this Messenger app certainly should be.

What version of OS X are you running?

**Comment by rsms** [comment]
I made a build with NSHighResolutionCapable=YES and screenshotted both the released 0.0.8 version and the one with the flag enabled.

Here's an animated GIF switching between the two screenshots at ~100ms intervals:

retina-vs-retina

Here're the two cut-out from the screenshot side-by-side:

retina-vs-retina

I certainly see no difference. (I'm using a 2015 MacBook pro 15" with no external display)

**Comment by rsms** [comment]
Release 0.0.8.2015-06-09T21:38:12Z includes this info dict property. Let me know if it works for you.

**Pull request closed without merging**